styleguide/source/assets/js/webforms.js

```diff
@@ -48,6 +48,12 @@
     }
   }
 
+  // Submits first page of Contact Us form on radio button selection
+  var $webform_buttons = $('#webform-submission-contact-us-add-form input[type="radio"]');
+  $webform_buttons.click(function(e) {
```

Comment:
I would bind the click.

`$( "#foo" ).bind( "click", function() {
  alert( "User clicked on 'foo.'" );
});`